### PR TITLE
[Fset-1424] Link Assessment Centre scores and feedback from candidate info page

### PR DESCRIPTION
**Pull request by mgilgar**
It also includes changes in fasttrack-admin-frontend and fasttrack-admin-acceptance-tests

### CODE REVIEW

File: app/services/testdata/StatusGeneratorFactory.scala [hunk 1]
```diff
@@ -48,6 +48,7 @@ object StatusGeneratorFactory {
       case ApplicationStatuses.AllocationUnconfirmed.name => AllocationStatusGenerator
       case ApplicationStatuses.FailedToAttend.name => FailedToAttendStatusGenerator
       case ApplicationStatuses.AssessmentScoresEntered.name => AssessmentScoresEnteredStatusGenerator
+      case "ASSESSMENT_SCORES_SUBMITTED" => AssessmentScoresSubmittedStatusGenerator
```
Comment (henricook) [inline app/services/testdata/StatusGeneratorFactory.scala]: This looks a little out of place, is it correct?

Comment (mgilgar) [inline app/services/testdata/StatusGeneratorFactory.scala]: I think is right, why do you say so?

Comment (mgilgar) [inline app/services/testdata/StatusGeneratorFactory.scala]: It is a pseudo status

Comment (henricook) [inline app/services/testdata/StatusGeneratorFactory.scala]: I think we should probably avoid using that? What status would the user actually be in when their scores are submitted?

Comment (henricook) [inline app/services/testdata/StatusGeneratorFactory.scala]: I've just read up on it. I'd probably make this an assessmentscoresentered generator with a submitted=true/false parameter - to save confusing anyone using the TDG

Comment (mgilgar) [inline app/services/testdata/StatusGeneratorFactory.scala]: I don't know, I really think we should have created a new status for that. And the TDG  and what is the "right" way to do it is a perfect example. Having to pass that as a parameter is a way to make complicated and to be honest in my case I would probably avoid using it if I have to go through that hassle.

Comment (henricook) [inline app/services/testdata/StatusGeneratorFactory.scala]: My issue is that someone would run this and reasonably expect a user to be in a _SUBMITTED status, and they could think it's not working - when in fact that status doesn't exist. Knowing when to break something up into a status is something that's not clearly defined, which i imagine is why we get the variation - but i'd respect the boundaries that have been drawn by the implementers rather than introducing inconsistency

Comment (mgilgar) [inline app/services/testdata/StatusGeneratorFactory.scala]: The whole status thing is a completely mess. Just to give a relevant example:
We have ENTERED status when all scores have been saved by QAKs (I assume) but for SUBMITTED, we rely on the data itself (three isSubmitted fields). What we have done for ENTERED, we should have done it for SUBMITTED or not do any of them.

Another idea that comes to my mind, is if ENTERED was not supposed to be the SUBMITTED status. Probably in the old fast track we did not have the separation between saving the scores and submitted and saving was "submitting". In the new FastTrack it does not make sense to have the ENTERED status because until they are not SUBMITTED we cannot make progress to next status (ACCEPTED).

Because of that when we want to create the data that is useful for testing, the data to test the edge cases, it happens that the TDG, that is based in the concept of application status, does not create what we need in an easy way, and the purpose of it was to be simple to use and to extend.

Probably that has happened because we did not do acceptance tests, which could have exposed this issue.

As this new status is not used by acceptance tests, I am going to remove it.
